src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/appointments/AppointmentsToNomisIntTest.kt

```diff
@@ -375,7 +378,9 @@ class AppointmentsToNomisIntTest : SqsIntegrationTestBase() {
           await untilAsserted {
             mappingServer.verify(
               2,
-              postRequestedFor(urlEqualTo("/mapping/appointments")),
+              postRequestedFor(urlEqualTo("/mapping/appointments"))
+                .withRequestBody(matchingJsonPath("appointmentInstanceId", equalTo("$APPOINTMENT_INSTANCE_ID")))
+                .withRequestBody(matchingJsonPath("nomisEventId", equalTo("$EVENT_ID"))),
```

Comment:
This test should have caught the bug but it wasn't checking the JSON to make sure the right details are being passed in the retry message.